packages/next/build/webpack/plugins/terser-webpack-plugin/src/TaskRunner.js

```diff
@@ -71,20 +58,19 @@ export default class TaskRunner {
 
     tasks.forEach((task, index) => {
       const cachePath = this.cacheDir && join(this.cacheDir, task.cacheKey)
-
-      const enqueue = () => {
-        this.boundWorkers(task, (error, data) => {
-          const result = error ? { error } : data
+      const enqueue = async () => {
+        try {
+          const result = await this.boundWorkers(task)
           const done = () => step(index, result)
-
-          if (this.cacheDir && !result.error) {
-            writeFileP(cachePath, JSON.stringify(data), 'utf8')
+          if (cachePath) {
+            writeFileP(cachePath, JSON.stringify(result), 'utf8')
               .then(done)
               .catch(done)
-          } else {
-            done()
           }
-        })
+        } catch (error) {
+          console.error('Error occurred minifying', error)
```

Comment:
```suggestion
```

We already handle this elsewhere, right? Was this for debugging?

Reply:
Yeah, I added it for debugging, although the error shown without this isn't very helpful so I left it in

Reply:
Hmm, the terser error should be pushed into the webpack errors -- are we sure the terser error is propagating correctly? Or what specific error is this for, an internal `jest-worker` error?

Reply:
Terser errors are propagated correctly it wasn't logging the errors I encountered when adding `jest-worker` so thought it would be good to log them there also but I guess it's not needed since that shouldn't happen, removed it